numpyro/distributions/util.py

```diff
@@ -623,6 +623,17 @@ def is_prng_key(key):
         return False
 
 
+def assert_one_of(**kwargs):
```

Comment:
Added this `assert_one_of` function because the pattern appears commonly across the code base.